fastai/optimizer.py

```diff
@@ -27,15 +39,27 @@ def freeze(self):
         assert(len(self.param_lists)>1)
         self.freeze_to(-1)
 
-    def set_freeze(self, n, rg, ignore_force_train=False):
+    def set_freeze(self,
```

Comment:
Missing docstring and some type annotations.

Reply:
Done